components/ee/ws-scheduler/pkg/scheduler/scheduler.go

```diff
@@ -224,7 +224,6 @@ func (s *Scheduler) startInformer(ctx context.Context) (schedulerQueue chan *cor
 			//		entry to early might lead to us making the same mistake again.
 			if pod.Spec.NodeName != "" && pod.Status.Phase != corev1.PodPending {
 				s.localBindingCache.delete(pod.Name)
-				log.WithField("pod", pod.Name).WithField("node", pod.Spec.NodeName).WithField("phase", pod.Status.Phase).Debug("removed from localBindingCache")
```

Comment:
That line was super helpful with debugging, actually... but if we hope we're fine with the current state we can remove.

Reply:
I figured :)

In operation though it's so spammy that it makes the logs pretty useless (other than for the debugging use-case). Maybe that's indicative for another problem?

If you're fine with it I'd like to clean up the logs a bit, and if we need this like back in for debugging, we can add it back. WDYT?